Make Stackalytics link advanced

alpineriveredge opened the pull request:
Commits page has commits link and reviews page has reviews one.

Commit:
Make Stackalytics link advanced

Commits page has commits link and reviews page has reviews one.

yuanying commented on lib/stchart/assets/table.html.erb:
```diff
@@ -37,7 +37,7 @@
       <td style="background-color: <%= color[0] %>;">&nbsp;</td>
       <td class='name'>
-        <a href="http://stackalytics.com/?user_id=<%=id%>">
+        <a href="http://stackalytics.com/?user_id=<%=id%>&metric=<%= metric == 'reviews' ? 'marks' : metric %>">
```
Could you use "&amp;" instead of "&" ?
- http://stackoverflow.com/questions/3705591/do-i-encode-ampersands-in-a-href

alpineriveredge replied:
Words inside "<%= %>" are ruby's conditional operator. So there are not rendered in HTML.

yuanying commented on lib/stchart/assets/template.html.erb:
```diff
@@ -173,7 +173,7 @@
           <td style="background-color: <%= color[0] %>;">&nbsp;</td>
           <td class='name'>
-            <a href="http://stackalytics.com/?user_id=<%=id%>">
+            <a href="http://stackalytics.com/?user_id=<%=id%>&metric=commits">
```
ditto

yuanying commented on lib/stchart/assets/template.html.erb:
```diff
@@ -203,7 +203,7 @@
           <td style="background-color: <%= color[0] %>;">&nbsp;</td>
           <td class='name'>
-            <a href="http://stackalytics.com/?user_id=<%=id%>">
+            <a href="http://stackalytics.com/?user_id=<%=id%>&metric=marks">
```
ditto

Commit:
Make Stackalytics link advanced

Commits page has commits link and reviews page has reviews one.
Replace "&" to "&amp;".

yuanying commented:
I think you should rebase this patch to compress one patch. If you send `pull request` to another.

alpineriveredge commented:
Thank you advice! I will rebase next time.
